### ISSUE TITLE
Improve accessibility/aria-labels on MenuSelect components

### ISSUE THREAD

**Issue by sjdemartini** [sjdemartini/mui-tiptap#175]
> would it be possible to add a aria-label to the inputs like font family and font size?  There currently is one, but it's not on the input so there is a missing label error from an accessibility standpoint

_Originally posted by @sgober in https://github.com/sjdemartini/mui-tiptap/issues/174#issuecomment-1785888733_

Based on https://mui.com/material-ui/react-select/#accessibility, it seems we perhaps shouldn't be providing an `aria-label` the way we are, though it's not entirely clear to me what the right pattern is when you don't have a separate label element. (https://mui.com/material-ui/react-select/#accessibility, https://mui.com/material-ui/react-text-field/#accessibility)
